test/e2e_node/node_problem_detector.go

```diff
@@ -21,9 +21,10 @@ import (
 	"os"
```

Comment:
I played around with computing the times in a more cross-platform way, but the end result was other failing node tests. The e2e package then failed on e2e_node_suite_test.go because darwin does not have /proc/cgroup support.